doc/platforms/x86_64_linux/x86_64_android.md

```diff
@@ -0,0 +1,113 @@
+...
+
+```
+for example:
+```
+$ ./build.sh object x86_64
+...
+
+[INFO]	Replacing existing vendor dependencies
+
+-----------------------------------
+ Build clean
+-----------------------------------
+go clean
+rm -rf /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/capi/output
+rm -rf /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/javaapi/output
+
+
+**********************************
+ Target Binary arch is amd64 
+**********************************
+
+----------------------------------------
+ Create Static object of Orchestration
+----------------------------------------
+mkdir -p /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/capi/output/inc/linux_x86-64 /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/capi/output/lib/linux_x86-64
+CGO_ENABLED=1 go build -ldflags '-extldflags "-static" -X main.version= -X main.commitID=70d67d1 -X main.buildTime=20200722.0008 -X main.buildTags=' -o /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/capi/output/lib/linux_x86-64/liborchestration.a -buildmode=c-archive interfaces/capi || exit 1
+mv /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/capi/output/lib/linux_x86-64/liborchestration.h /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/capi/output/inc/linux_x86-64/orchestration.h
+ls -al /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/capi/output/lib/linux_x86-64
+total 26764
+drwxrwxr-x 2 virtual-pc virtual-pc     4096 лип 22 00:09 .
+drwxrwxr-x 3 virtual-pc virtual-pc     4096 лип 22 00:08 ..
+-rw-rw-r-- 1 virtual-pc virtual-pc 27397614 лип 22 00:09 liborchestration.a
+
+**********************************
+ Target Binary is for Android 
+**********************************
+
+-------------------------------------------
+ Create Android archive from Java interface
+-------------------------------------------
+mkdir -p /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/javaapi/output
+gomobile init
+gomobile bind -ldflags '-X main.version= -X main.commitID=70d67d1 -X main.buildTime=20200722.0008 -X main.buildTags=' -o /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/javaapi/output/liborchestration.aar -target=android/amd64 -androidapi=23 interfaces/javaapi || exit 1
+ls -al /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/javaapi/output
+total 6316
+drwxrwxr-x 2 virtual-pc virtual-pc    4096 лип 22 00:09 .
+drwxrwxr-x 3 virtual-pc virtual-pc    4096 лип 22 00:09 ..
+-rw-rw-r-- 1 virtual-pc virtual-pc 6444465 лип 22 00:09 liborchestration.aar
+-rw-rw-r-- 1 virtual-pc virtual-pc    9996 лип 22 00:09 liborchestration-sources.jar
+
+
+**********************************
+ Edge-orchestration Archive 
+**********************************
+tree /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/capi/output
```

Comment:
Someone may need 'sudo apt-get install tree'

Reply:
Done. Thank you!